src/check/__tests__/check.test.js

```diff
@@ -80,4 +80,38 @@ describe("one-version: check", () => {
       });
     }).toThrow(FAILED_CHECK_ERROR);
   });
+
+  it("includes the manifest at -f in the check", () => {
+    const mockDetectPackageManager = jest.fn();
+    mockDetectPackageManager.mockReturnValue(packageManager);
+
+    const mockGetPackageDeps = jest.fn().mockReturnValue({});
+    const mockIsValidWorkspace = jest.fn().mockReturnValue(true);
+    const mockGetWorkspaces = jest.fn().mockReturnValue([]);
```

Comment:
Minor (optional) note: the only functions that need to be mocked with `jest.fn()` are any that you want to make assertions on. These could be removed and the call below could be:

```js
    check({
      file: "some-existent-path",
      getPackageManager: () => packageManager,
      getConfig: mockGetConfig,
      getDependencies: mockGetPackageDeps,
      getWorkspaces: () => [],
      validateWorkspace: () => true,
    });
```

Again, fine to leave in, it would just cut down on some unnecessary lines of code (without sacrificing readability IMO)